source/core/src/main/com/csse3200/game/rendering/Renderer.java

```diff
@@ -100,9 +100,6 @@ public void render() {
     batch.begin();
     renderService.render(batch);
     batch.end();
-    if (ServiceLocator.getLightService() != null) {
```

Comment:
This method is definitely needed for light rendering and not sure why this was removed?